suro-core/src/main/java/com/netflix/suro/sink/ThreadPoolQueuedSink.java

```diff
@@ -40,7 +40,7 @@ public boolean offer(Runnable runnable) {
                 maxPoolSize == 0 ? 10 : maxPoolSize,
                 10, TimeUnit.SECONDS,
                 jobQueue,
-                new ThreadFactoryBuilder().setNameFormat(threadFactoryName + "-Sender-%d").build());
+                new ThreadFactoryBuilder().setNameFormat(threadFactoryName + "-Sender-%d").setDaemon(true).build());
```

Comment:
This would cause data loss when the platform shuts down. May not be an issue for us but we need to be aware.